### DISCUSSION TITLE
deepmd-kit-2.1.2-cpu can't find pair style deepmd?

### DISCUSSION THREAD

**Discussion by Xian-qi** [deepmodeling/deepmd-kit#1792] [Q&A]
I installed [deepmd-kit-2.1.2-cpu-Linux-x86_64.sh](https://github.com/deepmodeling/deepmd-kit/releases/download/v2.1.2/deepmd-kit-2.1.2-cpu-Linux-x86_64.sh), but i find a error lammps can't find pair style deepmd ? I use lmp -h, can't see it on the list

**Comment by lnnbig** [comment]
I also found this bug. It seems that USER-DEEPMD has not been included in this version of LAMMPS.

**Reply by Lewis-YL** [reply]
Have you solved this problem?

**Comment by njzjz** [comment]
Please activate the conda environment first:
```sh
conda activate /path/to/deepmd-kit
```
For more information, read conda documentation [Activating an environment](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment).

**Reply by Xian-qi** [reply]
I installed deepmd-kit-2.1.2-cpu-Linux-x86_64.sh again. And then use this command "conda activate /path/to/deepmd-kit", but can't find pair style deepmd?

**Reply by Xian-qi** [reply]
I‘m sure the env is activated.

**Reply by njzjz** [reply]
What's the output of `echo $LAMMPS_PLUGIN_PATH` after the env is activated? Does LAMMPS log something like plugins loaded?

**Reply by Xian-qi** [reply]
echo $LAMMPS_PLUGIN_PATH : /XXXX/XX/deepmd-kit-2.1.2/deepmd-kit/lib/deepmd_lmp， and i found the Installed packages  not  include USER-DEEPMD when i use lmp -h

**Reply by njzjz** [reply]
You should be able to use `pair_style deepmd` when `LAMMPS_PLUGIN_PATH` is set.

The `lmp -h` message is maintained by LAMMPS and not what we maintain. We can't do anything on our side.

**Reply by Xian-qi** [reply]
Yes,  I can use deepmd now. Thank you for your help.

**Comment by Lewis-YL** [comment]
How do you solve the problem in the end?

**Comment by chtchelkatchev** [comment]
I got the same error in deepmd 2.2.3 installed from stand alone distribution. Conda activate has been done.
echo $LAMMPS_PLUGIN_PATH gives correct path /s/ls4/users/nms/SOFT_new_centos/deepmd-kit-2.2.3-cuda11.6_gpu/lib/deepmd_lmp
***************
Message when trying run lammps: 
  using 1 OpenMP thread(s) per MPI task 
Open of file /s/ls4/users/nms/SOFT_new_centos/deepmd-kit-2.2.3-cuda11.6_gpu/lib/deepmd_lmp/dpplugin.so failed: /s/ls4/users/nms/SOFT_new_centos/deepmd-kit-2.2.3-cuda11.6_gpu/lib/deepmd_lmp/dpplugin.so: undefined symbol: MPI_Bcast
Loaded 0 plugins from /s/ls4/users/nms/SOFT_new_centos/deepmd-kit-2.2.3-cuda11.6_gpu/lib/deepmd_lmp
Reading data file ...
  orthogonal box = (0 0 0) to (10.024128 10.024128 10.024128)
  1 by 1 by 1 MPI processor grid
  reading atoms ...
  512 atoms
 *****
ERROR: Unrecognized pair style 'deepmd' (src/force.cpp:271)

**Reply by njzjz** [reply]
Hi @chtchelkatchev, this error was reported in #2739 and has been fixed. Please re-download the files that were released yesterday.

![Screenshot from 2023-08-14 15-48-18](https://github.com/deepmodeling/deepmd-kit/assets/9496702/87e6571d-87d7-4331-b78e-349b0bee4791)

**Reply by chtchelkatchev** [reply]
Thank You! Everything works well now.
